content/php/concepts/string-functions/terms/ord/ord.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'ord()'
+Description: 'Returns an integer between 0 and 255 corresponding to the binary value of the first character of a string.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`ord()`** function takes a string as its only parameter and returns an integer between 0 and 255 based on the first character in that string.
+
+The integer returned is the decimal equivalent of the binary representation of the character. This binary representation conforms to the [ASCII standard](https://www.ascii-code.com/). Therefore, the integer returned is the ASCII decimal value of the first character in the parameter string.
```

Comment:
```suggestion
The integer returned is the decimal equivalent of the binary representation of the character. This binary representation only applies to single-byte encodings (ASCII, ISO-8859, Windows 1252).
```

Reply:
The statement that the binary representation "only applies" to certain encoding standards is unclear. What does it mean for a binary representation (e.g. 100 101) to apply to ASCII, for example? It is clearer to say that the binary representation was chosen so as to conform with these single-byte encoding standards.

Reply:
I put forward this wording because I think it's closer to the phrasing used in the PHP documentation. However, I'm not strongly committed to it. You can leave your original text.